src/config/Config.zig

```diff
@@ -1381,6 +1381,22 @@ keybind: Keybinds = .{},
 ///   * `end` - Insert the new tab at the end of the tab list.
 @"window-new-tab-position": WindowNewTabPosition = .current,
 
+/// If `true`, show a numeric icon on the tabs that indicate the position of the
+/// tab. Tabs are numbered starting at 1. On GTK the icons appear as a number
+/// inside a circle on the left side of the tab. On macOS the icons appear as
+/// `⌘N` on the right side of the tab.
```

Comment:
Note this isn't exactly correct for macOS. On macOS, the tab accessory label (as it's called) shows the keyboard shortcut bound to `goto_tab:N`. It has nothing to do with the numeric position of the tab except by relation to that binding _usually_ being mapped that way.

Reply:
Removed references to macOS.